lib/src/working_copy.rs

```diff
@@ -336,6 +337,10 @@ impl TreeState {
         let mut deleted_files: HashSet<_> = self.file_states.keys().cloned().collect();
         while !work.is_empty() {
             let (dir, disk_dir, git_ignore) = work.pop().unwrap();
+            let visit = sparse_matcher.visit(&dir);
```

Comment:
Variable only used on next line, right? Inline in `if`?

Reply:
Done